end-to-end-test/local/docker/setup_docker_containers.sh

```diff
@@ -0,0 +1,275 @@
+        backend_image_name="cbioportal/cbioportal:$BACKEND_BRANCH"
+    else
+        docker build https://github.com/$BACKEND_PROJECT_USERNAME/cbioportal.git#$BACKEND_BRANCH \
+            -f docker/web-and-data/Dockerfile \
+            -t $backend_image_name
+    fi
+
+    cd $curdir
+}
+
+run_cbioportal_container() {
+
+    # stop cbioportal container if running
+   (docker stop $E2E_CBIOPORTAL_HOST_NAME && docker rm $E2E_CBIOPORTAL_HOST_NAME) || true
+
+    # start cbioportal
+    # port 8081 opened for development in Local context
+    docker run -d --restart=always \
+        --name=$E2E_CBIOPORTAL_HOST_NAME \
+        --net=$DOCKER_NETWORK_NAME \
+        -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+        -p 8081:8080 \
+        $backend_image_name
+    
+    sleeptime=0
+    maxtime=180
+    while ! docker run --rm --net=$DOCKER_NETWORK_NAME $backend_image_name ping -c 1 "$E2E_CBIOPORTAL_HOST_NAME" &> /dev/null; do
+        echo Waiting for cbioportal to initialize...
+        sleeptime=$sleeptime+10
+        if (($sleeptime > $maxtime)); then 
+            echo Timeout reached. Terminating test!
+            exit 1
+        fi
+        sleep 10
+    done
+
+}
+
+load_studies_in_db() {
+
+    # import study_es_0 gene panels
+    docker run --rm \
+        --name=cbioportal-importer \
+        --net=$DOCKER_NETWORK_NAME \
+        -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+        $backend_image_name \
+        sh -c 'cd /cbioportal/core/src/main/scripts; for FILE in /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel*.txt; do ./importGenePanel.pl --data $FILE; done'
+
+    # import study_es_0
+    docker run --rm \
+        --name=cbioportal-importer \
+        --net=$DOCKER_NETWORK_NAME \
+        -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+        $backend_image_name \
+        python3 /cbioportal/core/src/main/scripts/importer/metaImport.py \
+        --url_server "http://$E2E_CBIOPORTAL_HOST_NAME:8080/cbioportal" \
+        --study_directory /cbioportal/core/src/test/scripts/test_data/study_es_0 \
+        --override_warning
+
+    # import custom studies
+    for DIR in "$TEST_HOME"/studies/*/; do
+
+        docker run --rm \
+            --name=cbioportal-importer \
+            --net=$DOCKER_NETWORK_NAME \
+            -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+            -v "$DIR:/study:ro" \
+            $backend_image_name \
+            python3 /cbioportal/core/src/main/scripts/importer/metaImport.py \
+            --url_server "http://$E2E_CBIOPORTAL_HOST_NAME:8080/cbioportal" \
+            --study_directory /study \
+            --override_warning
+    done
+
+}
+
+check_jitpack_download_frontend() {
```

Comment:
do need this function? Doesn't really hurt to keep I guess

Reply:
Eh no, we do not need it anymore. I have removed it.